evo_researcher/benchmark/utils.py

```diff
@@ -106,16 +129,21 @@ def _map_fields(old: dict[str, str], mapping: dict[str, str]) -> dict[str, str]:
 
 
 def get_polymarket_markets(
-    number: int = 100, excluded_questions: t.List[str] = []
+    number: int = 100, excluded_questions: t.List[str] = [], active: bool | None = True, closed: bool | None = False
 ) -> t.List[Market]:
-    if number > 100:
```

Comment:
It behaves differently if `closed` is not set, vs. if it's false or true.

So adding it to params only if it's not none, and removing this as it can return more markets than 100.